docs/usage.rst

```diff
@@ -69,6 +69,82 @@ callable is called, or to prevent it, use the ``call_default`` parameter:
     Note that ``call_default`` is only used when the related setting is missing
     from the project settings!
 
+Inherit from environment variables
+''''''''''''''''''''''''''''''''''
+
+Nowadays it became more and more popular to inherit settings from the environment.
+This functionality is supported as well. If the setting name is found in ``os.environ``,
+this value is used. To distinguish between different setting types, the value is
```

Comment:
It the setting is found in environment, its value is used and takes precedence over any value present in the settings module.

Reply:
Done.